src/otx/algorithms/detection/configs/base/data/tiling/efficientnet_iseg_tile_pipeline.py

```diff
@@ -8,7 +8,7 @@
 img_size = (512, 512)
 
 tile_cfg = dict(
-    tile_size=400, min_area_ratio=0.9, overlap_ratio=0.2, iou_threshold=0.45, max_per_img=1500, filter_empty_gt=True
+    tile_size=400, min_area_ratio=0.9, overlap_ratio=0.2, iou_threshold=0.7, max_per_img=1500, filter_empty_gt=True
```

Comment:
How do we know that 0.7 is better?

Reply:
This is for sync with models' parameter. I thought our plan is 
1. Sync model and tiler's nms iou threshold
2. Conduct exp for verifying it
@sovrasov @eugene123tw Feel free to correct me.

Reply:
That's right, the only problem is the default value.